operations/mimir-rules-action/README.md

```diff
@@ -12,6 +12,7 @@ This action is configured using environment variables defined in the workflow. T
 | `MIMIR_TENANT_ID`            | ID for the desired tenant in the target Mimir cluster. Used as the username under HTTP Basic authentication.                                                                                                                                                                                       | `false`  | N/A     |
 | `MIMIR_API_KEY`              | Optional password that is required for password-protected Mimir clusters. An encrypted [github secret](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets) is recommended. Used as the password under HTTP Basic authentication. | `false`  | N/A     |
 | `MIMIR_AUTH_TOKEN`           | Optional bearer or JWT token that is required for Mimir clusters authenticating by bearer or JWT token. An encrypted [github secret](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets) is recommended.                         | `false`  | N/A     |
+| `MIMIR_EXTRA_HEADERS`        | Newline-separated list of header=value pairs to add to the request.                                                                                                                                                                                                                                | `false`  | N/A     |
```

Comment:
In the description, I would clarify it's only supported by `mimirtool rules` command.

Reply:
Got it!